vizro-ai/hatch.toml

```diff
@@ -56,6 +57,7 @@ test-unit-coverage = [
   "- coverage combine",
   "coverage report"
 ]
+vizro-ai-ui = "python examples/dashboard_ui/app.py &"
```

Comment:
Is this hatch command really necessary? Could this not work like the other examples in `vizro-core` where the user can specify the example by providing `args`

Reply:
Couldn't do it. Let's take a look a this way together